core/src/main/scala/com/mobimeo/gtfs/model/ExtendedRouteType.scala

```diff
@@ -60,6 +60,8 @@ enum RouteType(val value: Int) extends IntEnumEntry {
   case CommuterCoachService      extends RouteType(208)
   case AllCoachServices          extends RouteType(209)
 
+  case AlfBusService extends RouteType(300)  // 300 special DemandAndResponse case for NRW ÖPNV
```

Comment:
Can we expand on this a bit and maybe link to the data set in question? And also describe more precisely what kind of route this is? 

If people not familiar with German abbreviations read this, they'll understand nothing. And even if they know the abbreviations, they'll have little clue about what kind of vehicle this is.